internal/actions/pr.go

```diff
@@ -265,6 +265,11 @@ func CreatePullRequest(
 		return nil, errors.Errorf("no commits between %q and %q", prCompareRef, opts.BranchName)
 	}
 
+	// Check if a parent branch has already been merged or not
+	if parentMeta.MergeCommit != "" {
+		return nil, errors.Errorf("failed to create a pull request. The parent branch %q has already been merged", parentMeta.Name)
```

Comment:
Can you mention that "Please run av stack sync to rebase the branch first."

Reply:
updated